mitodl/open-discussions
# Support for SAML (Touchstone) login

#### What are the relevant tickets?
- Partially addresses #671

#### What's this PR do?
- Add support for SAML login/registration (used by Touchstone)

#### How should this be manually tested?
- `docker-compose build` (new requirements)
- Follow the steps in the [new section of the README](https://github.com/mitodl/open-discussions/tree/mb/saml#7-testing-integration-with-saml-via-ssocircle)
- Depending on the email you registered with for SSOCircle, a new user with profile should be created or you will be logged in as an existing user with the matching email address.
